gsoc2018-bharat/src/lstm.py

```diff
@@ -0,0 +1,99 @@
+import torch
+import torch.autograd as autograd
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import numpy as np
+from sklearn.datasets.base import Bunch
+import json
+from gensim.models import FastText
+import logging
+import sys
+import argparse
+
+torch.manual_seed(1)
+
+logging.basicConfig(format='%(levelname)s : %(message)s', level=logging.INFO)
+logging.root.level = logging.INFO
+
+wv = None
+
+def load_model(savedModel):
+	"""
+	Load the saved model containing
+	pre-trained embeddings. I deleted
+	the model while just keeping the
+	vectors to save memory.
+	"""
+	global wv
+	model = FastText.load(savedModel)
+	wv = model.wv
+	del model
+
+def load_mappings(descriptions):
+	"""
+	Input to an LSTM must be a 3D tensor. For that, I
+	am reading all the descriptions line by line and
+	storing their embeddings row wise into a numpy
+	array of 3 dimensions and shape (n, 500, 300).
```

Comment:
no need to declare globals if your function just returns them

Reply:
Yes, will do that.